src/components/Codeblock.js

```diff
@@ -42,6 +42,16 @@ const LineContent = styled.span`
   display: table-cell;
 `
 
+const CopyEmoji = styled.div`
```

Comment:
What's the purpose of these divs? Why are their widths different?

Reply:
Looks like the spacing of your changes are off slightly: https://share.getcloudapp.com/WnurlkW2